src/main/java/com/smartling/marketo/sdk/rest/command/form/CreateFormRichTextField.java

```diff
@@ -0,0 +1,33 @@
+package com.smartling.marketo.sdk.rest.command.form;
+
+import com.smartling.marketo.sdk.domain.form.FormField;
+import com.smartling.marketo.sdk.rest.command.BaseMarketoCommand;
+
+import java.util.Collections;
+import java.util.Map;
+
+public class CreateFormRichTextField extends BaseMarketoCommand<FormField> {
+    private final int formId;
+    private final String text;
+
+    public CreateFormRichTextField(int formId, String text) {
+        super(FormField.class);
+        this.formId = formId;
+        this.text = text;
+    }
+
+    @Override
+    public String getPath() {
+        return "/asset/v1/form/" + formId + "/richText.json";
```

Comment:
Maybe better to use String::format?

Reply:
String::format is very slow and memory consuming, I only use it in exception handling

Reply:
Sounds like premature optimization which is not good.